pulp_smart_proxy/app/views.py

```diff
@@ -0,0 +1,69 @@
+from urllib.parse import urljoin
+
+from django.conf import settings
+from drf_spectacular.utils import extend_schema
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from pulpcore.app.apps import pulp_plugin_configs, get_plugin_config
+
+
+class FeaturesView(APIView):
+    """
+    Returns features of the smart_proxy
+    """
+
+    @extend_schema(
+        summary="Inspect features",
+        operation_id="features_read",
+    )
+    def get(self, request):
+        data = ["pulpcore"]
+        return Response(data)
+
+
+class FeaturesV2View(APIView):
+    """
+    Returns features of the smart_proxy in v2 format
+    """
+
+    @extend_schema(
+        summary="Inspect features",
+        operation_id="featuresv2_read",
+    )
+    def get(self, request):
+        host = settings.CONTENT_ORIGIN or request.get_host()
```

Comment:
This should be the API URL, not content. It should also be a URL, so include the protocol and port. I think it should be something like `request.build_absolute_uri(settings.V3_API_ROOT)`.

Reply:
Yes, but Katello (the only consumer of what we're building here) already appends `/pulp/api/v3` :see_no_evil: 

https://github.com/Katello/katello/blob/a739b7104bde511b2a77eead93c6207cc94943a2/app/models/katello/concerns/smart_proxy_extensions.rb#L406-L409